src/components/common.js

```diff
@@ -173,22 +173,28 @@ export const pageColumn = function(title, flex, contents) {
     ])
 }
 
-export const spinnerOverlay = div(
+export const spinnerDefault = ({ outerStyles = {}, innerStyles = {} }) => div(
```

Comment:
why spinnerDefault instead of just spinner?

Reply:
Right, but the build on it, so these are the base settings, which is why I was thinking default